### PR TITLE
Updated .ipa page

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/automated-testing/ipa-files.md [hunk 1]
```diff
@@ -8,23 +8,18 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 If you need to maintain your own provisioning profile in the .ipa file, disable the **App Resigning** option when you set the Device Settings for your app in the cloud.
 
 You can also use the Appium capability `resigningEnabled=false` to disable app resigning. This option is available for private devices only.
-:::
 
-### Building an .app Bundle
+#### Building an .app Bundle
 1. Open your application project in Xcode.
 2. Select your product's device target:
     1. On a real device: **Generic iOS Device** or **Any iOS Device (arm64)**.
```
Comment (enriquegh) [inline docs/mobile-apps/automated-testing/ipa-files.md:25]: We should remove the option of setting a Sim device target. That is not needed.

File: docs/mobile-apps/automated-testing/ipa-files.md [hunk 2]
```diff
@@ -86,3 +81,14 @@ To set the iOS version in your Xcode Target:
 :::caution
 This will also overwrite the **Build Settings** at the Project level to that iOS version. If you use this method, be aware that your Targets can become out of sync with each other and the Project settings, and your tests will break. If you change the iOS version for one target output, you may want to build the Project again to make sure all your targets are in sync.
 :::
+
+## Virtual Devices
+### Building an .app Bundle
+1. Open your application project in Xcode.
+2. Select your product's device target:
+    1. On a real device: **Generic iOS Device** or **Any iOS Device (arm64)**.
```
Comment (enriquegh) [inline docs/mobile-apps/automated-testing/ipa-files.md:89]: Same as above, a real device target is not needed when building for Simulators.

**Review by enriquegh — CHANGES_REQUESTED** [review]
Looks good! Just two places to remove a step but other than that looks good.

**Review by enriquegh — APPROVED** [review]

**Comment by enriquegh** [comment]
LGTM

File: docs/mobile-apps/automated-testing/ipa-files.md [hunk 3]
```diff
@@ -8,38 +8,31 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This topic covers:
-* For Appium: How to create an iOS .ipa file using self-signed application files.
-* For XCUITest: How to create iOS .app files with our XCUITest Real Device Test Runner and Virtual Device Test Runner.
-
 We use a resigning process to embed our own provisioning profile into the uploaded .ipa file. With your own profile in place, the app will be allowed to install and launch on cloud devices. The functionality of the app itself remains completely unmodified.
```
Comment (sweeneyskirt-sl) [inline docs/mobile-apps/automated-testing/ipa-files.md:11]: I know you didn't edit this paragraph, but is `your` a typo in the second sentence? It doesn't make sense to me --  _"We use...our own provisioning profile..." With **your** own profile in place...."_

Are we saying that the ipa need 2 profiles? Ours AND yours?

**Review by sweeneyskirt-sl — COMMENTED** [review]
All your changes look good, @lauramthomas . I just had one question about the opening paragraph. I *think* it's a typo, but I am not sure.

File: docs/mobile-apps/automated-testing/ipa-files.md (hunk 3 above)
Comment (lauramthomas) [reply docs/mobile-apps/automated-testing/ipa-files.md]: I changed "your own" to "the." I think that should solve/sidestep the issue so I can get this PR merged.

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**